components/hooks/useDbView.tsx

```diff
@@ -1,42 +1,43 @@
-import { useEffect } from "react";
+import { useEffect, useMemo } from "react";
 import { useRouter } from "next/router";
-import { atom, useSetAtom } from "jotai";
-import { atomWithStorage } from "jotai/utils";
-
-// eslint-disable-next-line no-shadow
-export enum DbViewEnum {
-  GRAPH = "graph",
-  NUMBERS = "numbers",
-  TABLE = "table",
-  TEXT = "text",
-}
-
-export const currentViewAtom = atom<DbViewEnum>(DbViewEnum.TEXT);
-export const shouldShowSegmentNumbersAtom = atomWithStorage<boolean>(
-  "shouldShowSegmentNumbers",
-  true,
-);
-export const shouldUseOldSegmentColorsAtom = atomWithStorage<boolean>(
-  "shouldUseOldSegmentColors",
-  true,
-);
-
-const initiateView = (view: DbViewEnum | string): DbViewEnum => {
-  if (Object.values(DbViewEnum).includes(view as DbViewEnum)) {
-    return view as DbViewEnum;
-  }
-  return DbViewEnum.TEXT;
+import { currentViewAtom } from "features/atoms";
+import { DbViewEnum } from "features/sidebarSuite/config/types";
+import { useSetAtom } from "jotai";
+
+import { useDbQueryParams } from "./useDbQueryParams";
+
+export const isValideView = (view: unknown): view is DbViewEnum =>
+  Object.values(DbViewEnum).some((item) => item === view);
+
+export const getSafeView = (view: unknown) => {
+  return isValideView(view) ? view : DbViewEnum.TEXT;
```

Comment:
maybe we can export a DEFAULT_VIEW constant from somewhere so that it can be changed in one place later if needed?